README.md

```diff
@@ -1 +1,35 @@
-# liberty-backstage-demo
+# Open Liberty Backstage Template
+This is a sample template for Backstage/Red Hat Developer Hub that creates and deploys the Liberty Starter app on Kubernetes.  Though this template may work as is, the best practice for templates in Backstage to be designed by Platform Engineers for their specific infrastructure.  This template should be used as a base for designing your own template in Backstage.  
```

Comment:
Should this be something like this? - "Though this template may work as is, the best practice is for Platform Engineers to design templates specific to their infrastructure."